### PR TITLE
pass the options parameter in the link component

### PR DESCRIPTION
**Pull request by Ivan96318**
Some links with the  disabled option did not take effect because the options parameter was not passed to the link component inside the SideMenu::Item::Component

Before:
Captura de Pantalla 2022-06-29 a la(s) 14 58 12

After:
Captura de Pantalla 2022-06-29 a la(s) 14 57 34

### CODE REVIEW

File: spec/components/bali/side_menu_spec.rb [hunk 1]
```diff
@@ -54,4 +54,14 @@
 
     expect(rendered_component).to have_css 'a.is-active', text: 'item'
   end
+
+  it 'renders a disabled link' do
```
Comment (alexisKinto) [inline spec/components/bali/side_menu_spec.rb:58]: How about adding this case in the side_menu preview too?

File: Gemfile.lock [hunk 2]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    bali_view_components (0.14.0)
+    bali_view_components (0.14.1)
```
Comment (rafaelcdelgado) [inline Gemfile.lock:4]: Don't forget to update this version,

File: spec/components/bali/side_menu_spec.rb (hunk 1 above)
Comment (rafaelcdelgado) [reply spec/components/bali/side_menu_spec.rb:58]: i'm down for it.

File: Gemfile.lock (hunk 2 above)
Comment (Ivan96318) [reply Gemfile.lock:4]: I updated the version already 🤔

**Review by fedegl — APPROVED** [review]

File: Gemfile.lock (hunk 2 above)
Comment (fedegl) [reply Gemfile.lock:4]: you need to merge main first and then update. Main is on 0.15.0 already

Comment (Ivan96318) [reply Gemfile.lock:4]: understood!

**Review by alexisKinto — APPROVED** [review]

**Pull request merged**